pkg/trtl/peers_test.go

```diff
@@ -0,0 +1,131 @@
+package trtl_test
+
+import (
+	"context"
+
+	"github.com/trisacrypto/directory/pkg/trtl/peers/v1"
+)
+
+// TestPeers is a unified function that performs multiple interactions with the Peers
+// server, getting, adding, and removing peers to exercise all Peers server methods
+// without requiring a dedicated test fixture.
+func (s *trtlTestSuite) TestPeers() {
+	// Setup the test and create a new Peers client
+	require := s.Require()
+	ctx := context.Background()
+
+	// Start the gRPC client.
+	require.NoError(s.grpc.Connect())
+	defer s.grpc.Close()
+	client := peers.NewPeerManagementClient(s.grpc.Conn)
+
+	// GetPeers should return nothing
+	rep, err := client.GetPeers(ctx, &peers.PeersFilter{})
+	require.NoError(err, "could not get empty peers")
+	require.Len(rep.Peers, 0, "unexpected peers returned, have the fixtures changed?")
+	require.Equal(int64(0), rep.Status.NetworkSize, "unexpected network size returned")
+	require.Len(rep.Status.Regions, 0, "unexpected regions mapping returned")
+
+	// Add several peers in different regions
+	network := []*peers.Peer{
+		{
+			Id:     1,
+			Addr:   "tc1.trtl.dev:443",
+			Name:   "alpha",
+			Region: "tauceti",
+		},
+		{
+			Id:     2,
+			Addr:   "tc2.trtl.dev:443",
+			Name:   "bravo",
+			Region: "tauceti",
+		},
+		{
+			Id:     3,
+			Addr:   "ei.trtl.dev:443",
+			Name:   "charlie",
+			Region: "epsiloniridini",
+		},
+		{
+			Id:     4,
+			Addr:   "wolf.trtl.dev:443",
+			Name:   "delta",
+			Region: "wolf579",
```

Comment:
Not going with the star trek [reference](https://memory-alpha.fandom.com/wiki/Battle_of_Wolf_359)?

Reply:
Whoops -- that's actually what I was going for; I'll update it to the correct wolf!